documentation/api.md

```diff
@@ -652,6 +652,56 @@ message | String | (Required) Comment contents as raw Markdown text. New lines r
 #### Response
 No `data` is returned.
 
+##View a Backup's Comments
+>><h4> Example Response </h4>
+> ```
+python
+import requests
+url = "https://okpy.org/api/v3/backups/a24x23/comment?access_token={}"
+access_token = 'test'
+{
+    "data": [
```

Comment:
This entire block to define data is un-needed right? It's not used in the requests.get call